src/pkg/resourceviews/resourcegroups.go

```diff
@@ -84,6 +86,28 @@ func (r *ResourceGroupListView) SpawnResourceTypeListView() tview.Primitive {
 	return rtList.List
 }
 
+func (r *ResourceGroupListView) SpawnVirtualMachineListView() tview.Primitive {
```

Comment:
This was replaced by the general resource view and both vm and aks specific views are not used

Reply:
Also made it so that you get the vm and aks specific views when clicking on the resource type.